data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg

```diff
@@ -1067,6 +1067,11 @@
                     message= _ "... the road ahead is clear and I am eager to blast some more orcs. Shall we proceed?"
                 [/message]
 
+                [message]
+                    speaker="Camerin"
```

Comment:
I can't remember if it's possible for Camerin to be killed - I think he can be. Although, I recall that if a unit with the `speaker` ID is not present then the text is skipped but it might be a good idea to wrap his line with a condition check.

May be worth checking other Camerin statements to see how those scenarios handle it.

Reply:
He can die (Which is sad) but everywhere else it's like that.